packages/drivers/odsp-driver/src/createOdspCreateContainerRequest.ts

```diff
@@ -20,7 +20,51 @@ import { buildOdspShareLinkReqParams, getContainerPackageName } from "./odspUtil
  * @param fileName - name of the new file to be created
  * @param createShareLinkType - type of sharing link you would like to create for this file. ShareLinkTypes
  * will be deprecated soon, so for any new implementation please provide createShareLinkType of type ShareLink
- * @param containerPackageInfo - container package information which will be used to extract the container package name.
+ * @legacy
+ * @alpha
+ */
+export function createOdspCreateContainerRequest(
+	siteUrl: string,
+	driveId: string,
+	filePath: string,
+	fileName: string,
+	createShareLinkType?: ISharingLinkKind,
+): IRequest;
+
+/**
+ * Overloaded version of createOdspCreateContainerRequest that takes in containerPackageInfo
```

Comment:
Just use the same basic description.
```suggestion
/**
 * Create the request object with url and headers for creating a new file on OneDrive Sharepoint
```

Reply:
Applied change